src/aat/resources/features/F-045/S-151.td.json

```diff
@@ -0,0 +1,27 @@
+{
+	"title": "must return 201 if the grant is successful for a user to a valid case ID",
+	"_guid_": "S-151",
+	"_extends_": "F-045_Test_Data_Base",
+
+	"specs": [
+		"a detailed profile in CCD",
+		"contains as input a valid case-reference that exists in CCD",
+		"has the 201 return code"
+	],
+  "expectedResponse": {
+
+    "responseCode" : 201,
+    "responseMessage" : "Created",
+    "headers" : {
+      "Cache-Control" : "no-cache, no-store, max-age=0, must-revalidate",
+      "Content-Length" : "[[ANY]]",
+      "Date" : "[[ANY]]",
+      "Expires" : "0",
+      "Pragma" : "no-cache",
+      "Vary" : "Accept-Encoding",
+      "X-Content-Type-Options" : "nosniff",
+      "X-Frame-Options" : "DENY",
+      "X-Xss-Protection" : "1; mode=block"
+    }
```

Comment:
a lot of this could come from a parent object

Reply:
There is no body in this 201 response which comes by default with parent object. Secondly "Content-Encoding" and "Content-Type" fields from common headers are not appearing in this response.